feature.py

```diff
@@ -67,57 +67,75 @@ def forward(self, x: th.Tensor) -> th.Tensor:
 class Residual(nn.Module):
     """CBA + Residual Block."""
 
-    def __init__(self, base: nn.Module):
-        self.base = base
+    def __init__(
+        self, 
+        c_in: int, 
+        c_out: int, 
+        padding: Union[int, Tuple[int, int]] = 1, 
+        stride: Union[int, Tuple[int, int]] = 1
+    ):
+        super().__init__()
+        self.conv1 = CBA(c_in, c_out, padding=padding, stride=stride)
+        self.conv2 = CBA(c_out, c_out, lrelu_eps=None)
+        self.residual = nn.Sequential()
```

Comment:
Does just using `nn.Sequential()` actually work, when the below `if` block evaluated to `False`?

Reply:
As far as I know (and it should be, since the first residual block uses empty nn.Sequential()), it works like an identity layer.